avm/res/edge/site/rg-scope/main.bicep

```diff
@@ -0,0 +1,76 @@
+metadata name = 'Microsoft Edge Site Resources'
+metadata description = 'Resource group scoped resources for Microsoft Edge Site.'
+
+// ============== //
+//   Parameters   //
+// ============== //
+
+@description('Required. Name of the resource to create.')
+param name string
+
+@description('Optional. Location for all Resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Enable/Disable usage telemetry for module.')
+param enableTelemetry bool = true
+
+@description('Optional. The description of the site.')
+param siteDescription string?
+
+@description('Optional. The display name of the site.')
+param displayName string = name
+
+@description('Optional. Labels for the site.')
+param labels resourceInput<'Microsoft.Edge/sites@2025-03-01-preview'>.properties.labels?
+
+@description('Required. The physical address configuration of the site.')
+param siteAddress resourceInput<'Microsoft.Edge/sites@2025-03-01-preview'>.properties.siteAddress
+
+// ============== //
+// Resources      //
+// ============== //
+
+#disable-next-line no-deployments-resources
+resource avmTelemetry 'Microsoft.Resources/deployments@2024-03-01' = if (enableTelemetry) {
+  name: '46d3xbcp.res.edge-site.${replace('-..--..-', '.', '-')}.${substring(uniqueString(deployment().name, location), 0, 4)}'
```

Comment:
Will need to be updated to final value once module is onboarded to MAR.
cc: @jeanchg & @matebarabas

Reply:
MAR PR: https://github.com/microsoft/mcr/pull/4366
Module Index PR: https://github.com/Azure/Azure-Verified-Modules/pull/2269